pkgs/development/libraries/libqmlbind/default.nix

```diff
@@ -49,6 +48,6 @@ stdenv.mkDerivation rec {
     homepage = https://github.com/seanchas116/libqmlbind;
     description = "C bindings for QML, used for binding to other languages";
     license = with licenses; [ mit fdl13 ];
-    maintainers = [ "0BobTheJanitor <Changing Email>" ]
+    maintainers = [ "0BobTheJanitor <Changing Email>" ];
```

Comment:
I'm currently getting a new email and domain, but I can add it now and just update it later.